src/core_plugins/interpreter/server/functions/timelion.js

```diff
@@ -1,12 +1,25 @@
 /*
```

Comment:
Why did this file move out of `canvas_plugin_src`?

Reply:
we plan to use it (for timelion purposes) in oss as well